### PR TITLE
🐛 Fix upload in shared with me directory

### LINKED ISSUE #520: [Bug] Upload does not work in shared with me directory
**Describe the bug**
Given a folder that is shared with me with "Full access" permission, I still can't upload/create anything within.

**To Reproduce**
1. User A creates a folder "Test" and shares it with another user B
2. User B goes to shared with me -> Test
3. User B can't see the "More" button for folder/file upload/create.

**Expected behavior**
A user with "Full access" permission should be able to upload/create files/folders within a shared folder.

### PR DESCRIPTION
**Pull request by MontaGhanmy**

### CODE REVIEW

File: tdrive/frontend/src/app/views/client/body/drive/browser.tsx [hunk 1]
```diff
@@ -359,41 +363,41 @@ export default memo(
...
-                {viewId !== 'shared_with_me' && (
-                  <Menu menu={() => onBuildContextMenu(details)}>
-                    {' '}
-                    <Button theme="secondary" className="ml-4 flex flex-row items-center">
-                      <span>
-                        {selectedCount > 1
-                          ? `${selectedCount} items`
-                          : Languages.t('scenes.app.drive.context_menu')}{' '}
-                      </span>
+                <Menu menu={() => onBuildContextMenu(details)}>
```
Comment (shepilov) [inline tdrive/frontend/src/app/views/client/body/drive/browser.tsx]: Why did you removed "viewId !== 'shared_with_me'" from this menu item?

Comment (MontaGhanmy) [reply tdrive/frontend/src/app/views/client/body/drive/browser.tsx]: To enable users to upload via the option in that context menu. Should I keep it disabled @shepilov ?

Comment (shepilov) [reply tdrive/frontend/src/app/views/client/body/drive/browser.tsx]: yes, you can't upload you you inside "shared_with_me"
and it's somehow connected with the display of the "Shared Drive"?

Comment (MontaGhanmy) [reply tdrive/frontend/src/app/views/client/body/drive/browser.tsx]: Yes the browser component is used across all the views, that's how it's connected.

**Review by shepilov — APPROVED** [review]

**Pull request merged**